src/components/Home/FeedEventFilters.tsx

```diff
@@ -0,0 +1,97 @@
+import { InputCheckbox } from '@components/UI/InputCheckbox';
+import { Tooltip } from '@components/UI/Tooltip';
+import { Menu, Transition } from '@headlessui/react';
+import { AdjustmentsIcon } from '@heroicons/react/outline';
+import clsx from 'clsx';
+import type { ChangeEvent } from 'react';
+import { Fragment } from 'react';
+import React from 'react';
+import { useTimelinePersistStore } from 'src/store/timeline';
+
+const FeedEventFilters = () => {
+  const feedEventFilters = useTimelinePersistStore((state) => state.feedEventFilters);
+  const setFeedEventFilters = useTimelinePersistStore((state) => state.setFeedEventFilters);
+
+  const handleChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setFeedEventFilters({ ...feedEventFilters, [e.target.name]: e.target.checked });
+  };
+
+  return (
+    <div className="flex justify-end">
+      <div className="flex items-center space-x-4">
```

Comment:
**question:** why flex for single children?

Reply:
clear clear